opt/with_jmap

```diff
@@ -5,4 +5,4 @@ export PATH=$HOME/.heroku/with_jmap/bin:$PATH
 export HEROKU_JAVA_HOME=${JAVA_HOME}
 unset JAVA_HOME
 
-eval "$@"
+"$@"
```

Comment:
This and the others should probably use `exec`, so that signals are propagated properly?

Reply:
Probably, yeah. I'm hesitant to change these though and I won't port them over to the CNBs. I'd rather keep them "as-is" (minus the ShellCheck fixes).